.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+**Screenshots**
+
+**Related Issue**
+E.g: *Closes/Fixes #1234*
+
+- [ ]  **Code Quality**
+    - [ ]  The code follows established coding standards and style guidelines
```

Comment:
It'd be nice to have a link to some guidelines. 
I think foundation had one we could use.

Reply:
We have some guidelines enforced by eslint, but I agree it would be good to have those standards and guidelines published somewhere

Reply:
I'm not sure about Foundation guidelines but we could find one that fits our coding style and conventions.

Reply:
Yeah, maybe the when and where for the FF should come with the issue definition, and here we just check if it's included in the PR

Reply:
Until we agree on a coding standard, let's remove this